edge/conf/edge.yaml

```diff
@@ -44,7 +44,7 @@ edged:
     remote-runtime-endpoint: unix:///var/run/dockershim.sock
     remote-image-endpoint: unix:///var/run/dockershim.sock
     runtime-request-timeout: 2
-    podsandbox-image: k8s.gcr.io/pause
+    podsandbox-image: kubeedge/pause:3.1 # kubeedge/pause:3.1 for x86 arch , kubeedge/pause-arm:3.1 for arm arch, kubeedge/pause-arm64 for arm64 arch
```

Comment:
now kubeedge/pause:3.1 not exist. we need to push `pause:3.1` to kubeedge
@kevin-wangzefeng @fisherxu 

![image](https://user-images.githubusercontent.com/36974756/64514288-ec638180-d31c-11e9-9787-881a62d191e9.png)

Reply:
Thanks @kadisi , I have pushed the `pause:3.1` image.